src/org/labkey/test/TestScrubber.java

```diff
@@ -140,6 +140,16 @@ public void cleanSiteSettings()
             TestLogger.error("Failed to reset active user limit after test", e);
         }
 
+        try
+        {
+            new SimplePostCommand("admin", "resetProperties")
+                    .execute(createDefaultConnection(), "/");
```

Comment:
Move this to a static method in `BaseSettingsPage` to be shared with test.

Reply:
Done.